### PR TITLE
swift-mesonlsp: Add version 3.1.3

### LINKED ISSUE #11988: [Request]: Swift-MesonLSP
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [ ] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

swift-mesonlsp

### Description

An unofficial, unendorsed language server for meson written in Swift.

### Homepage

https://github.com/JCWasmx86/Swift-MesonLSP

### Download Link(s)

https://github.com/JCWasmx86/Swift-MesonLSP/releases/download/v2.4.4/Swift-MesonLSP-win64.zip

### Some Indication of Popularity/Repute

Though its GitHub star count does not reach the requirement, it is the default LSP backend used by Meson's official [vscode-meson](https://github.com/mesonbuild/vscode-meson) editor plugin.

### PR DESCRIPTION
**Pull request by shanoaice**
Closes #11988.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by shanoaice** [comment]
Ooh, there's been an update, let me fix this.

File: bucket/swift-mesonlsp.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "2.4.4",
+    "description": "An unofficial, unendorsed language server for meson written in Swift.",
+    "homepage": "https://github.com/JCWasmx86/Swift-MesonLSP#readme",
```
Comment (aliesbelik) [inline bucket/swift-mesonlsp.json:4]:
```suggestion
    "homepage": "https://github.com/JCWasmx86/Swift-MesonLSP",
```

File: bucket/swift-mesonlsp.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/JCWasmx86/Swift-MesonLSP/releases/download/v2.4.4/Swift-MesonLSP-win64.zip",
+            "hash": "81e4c433623b344d8e321a0e48426b3f83844b108d099a8cac6993c546084b66"
+        }
+    },
+    "bin": "Swift-MesonLSP.exe",
+    "checkver": {
+        "github": "https://github.com/JCWasmx86/Swift-MesonLSP"
+    },
```
Comment (aliesbelik) [inline bucket/swift-mesonlsp.json:15]:
```suggestion
    "checkver": "github",
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Please update to the latest version as well.

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**